helix-core/src/main/java/org/apache/helix/manager/zk/ZKUtil.java

```diff
@@ -139,6 +139,7 @@ public static boolean isInstanceSetup(HelixZkClient zkclient, String clusterName
       requiredPaths.add(PropertyPathBuilder.instanceConfig(clusterName, instanceName));
       requiredPaths.add(PropertyPathBuilder.instanceMessage(clusterName, instanceName));
       requiredPaths.add(PropertyPathBuilder.instanceCurrentState(clusterName, instanceName));
+      requiredPaths.add(PropertyPathBuilder.instanceCustomizedState(clusterName, instanceName));
```

Comment:
Are we sure this is required path? That's not a good idea to add this. If we add this and existing product uses Helix library for participant will have check failure since Instance is already created.